api/capi/src/nnstreamer-capi-util.c

```diff
@@ -868,8 +868,34 @@ ml_validate_model_file (const char *model, ml_nnfw_type_e * nnfw)
   switch (*nnfw) {
     case ML_NNFW_TYPE_ANY:
       if (g_str_has_suffix (path_down, ".tflite")) {
-        ml_logi ("The given model [%s] is supposed a tensorflow-lite model.", model);
-        *nnfw = ML_NNFW_TYPE_TENSORFLOW_LITE;
+        /**
+         * .tflite is supported by both tensorflow and nnfw.
+         * Priority decided with ini file.
+         */
+        bool available = FALSE;
+
+#ifdef NNFW_RUNTIME_PRIORITIZE
```

Comment:
I don't recommend to use preprocessors for such decisions. 
It's better with nnstreamer configurations (.ini + envvar + hardcoded-defaults)

Reply:
> It's better with nnstreamer configurations (envvar + .ini + hardcoded-defaults)

Updated to this format.